cg/apps/mip/parse_sampleinfo.py

```diff
@@ -1,71 +1,82 @@
-from typing import List
+from typing import Dict, List, Literal
 
+from pydantic import BaseModel, EmailStr, parse_obj_as
 import yaml
 from cg.apps.tb.models import TrailblazerAnalysis
 
 
-def get_sample_data_from_config(config: dict) -> List[dict]:
+class MipBaseConfig(BaseModel):
+    """This model is used when validating the mip analysis config"""
+
+    analysis_type: Dict[str, str]
```

Comment:
Great that you are exposing the data like this 🥳 !!
I think that this should be it's own pydantic model so that we expose what the analysis type is even more.

Reply:
```
class AnalysisType(BaseModel):
    something1: Literal["hello", "you"]
    something2: Literal["happy", "chap"]
```

Reply:
```suggestion
    analysis_type: AnalysisType
```

Reply:
Done